Loop/Models/RemoteCommand.swift

```diff
@@ -64,11 +64,19 @@ extension RemoteCommand {
         }  else if let bolusValue = notification["bolus-entry"] as? Double {
             return .success(.bolusEntry(bolusValue))
         } else if let carbsValue = notification["carbs-entry"] as? Double {
-            // TODO: get default absorption value
-            var absorptionTime = TimeInterval(hours: 3.0)
-            if let absorptionOverride = notification["absorption-time"] as? Double {
-                absorptionTime = TimeInterval(hours: absorptionOverride)
+
+            let minAbsorptionTime = TimeInterval(hours: 0.5)
+            let maxAbsorptionTime = TimeInterval(hours: 8)
```

Comment:
This constant is repeated in CarbEntryViewController; please extract into LoopConstants

Reply:
Good idea - Done.